src/patternfly/components/Page/page.scss

```diff
@@ -90,6 +90,10 @@
   --pf-c-page__main-section--md--PaddingRight: var(--pf-global--spacer--xl);
   --pf-c-page__main-section--md--PaddingBottom: var(--pf-global--spacer--xl);
   --pf-c-page__main-section--md--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-page__main-section--md--m-no-padding-mobile--PaddingTop: 0;
```

Comment:
This looks great. Can you change the vars so that `--md` is the last thing before the property? The breakpoint should go last (assuming there aren't properties defined after the breakpoint in the SCSS). The list of things in the var should also generally follow the nesting in SCSS.